frontend/nuxt.config.ts

```diff
@@ -45,27 +47,22 @@ export default defineNuxtConfig({
 		'~/components/Drawer',
 		'~/components/BlockDetails',
 	],
-
-	runtimeConfig: {
-		public: {
-			...getConfig(ENVIRONMENT),
-			version: process.env.npm_package_version,
-			environment: ENVIRONMENT,
-			includeDevTools:
-				ENVIRONMENT === 'dev' ||
-				ENVIRONMENT === 'stagenet' ||
-				ENVIRONMENT === 'testnet',
-		},
-	},
-
+	// Global CSS files
 	css: ['~/assets/css/styles.css'],
-
+	// Enable postCSS
 	postcss: {
 		plugins: {
 			tailwindcss: {},
 			autoprefixer: {},
 		},
 	},
-
+	// Lock default values for Nuxt to what they were at this date.
 	compatibilityDate: '2024-11-01',
+	// TypeScript configurations.
+	// typescript: {
+	// 	// Enable strict checks.
+	// 	strict: true,
+	// 	// Enable type-checking at build time.
+	// 	typeCheck: true,
+	// },
```

Comment:
Should the comment be removed?

Reply:
I enabled it and then there were a lot of type errors, I have addressed these in a [separate PR](https://github.com/Concordium/concordium-scan/pull/291)